packages/gdl-frontend/components/EditBookLibrary/EditBooks.js

```diff
@@ -0,0 +1,237 @@
+        />
+      </EditBooksBar>
+    </Hidden>
+    <Hidden xsDown>
+      <EditBooksBar>
+        <div
+          style={{
+            width: 'calc(100% - 40px)',
+            position: 'absolute',
+            display: 'flex',
+            justifyContent: 'center'
+          }}
+        >
+          <Typography variant="h5" component="h5">
+            Edit {favorites ? 'favorites' : 'offline library'}
+          </Typography>
+        </div>
+        <IconButton
+          onClick={deselectAllBooks}
+          css={iconButtonStyle}
+          label=""
+          icon={
+            <Tooltip
+              title={
+                <FormattedMessage
+                  id="Deselect all"
+                  defaultMessage="Deselect all"
+                />
+              }
+            >
+              <Clear
+                color={selectedBooks.length === 0 ? 'disabled' : 'inherit'}
+              />
+            </Tooltip>
+          }
+        />
+        <Typography variant="subtitle1" component="p">
+          {selectedBooks.length} books chosen
+        </Typography>
+
+        <Right>
+          <IconButton
+            css={iconButtonStyle}
+            aria-label="select"
+            onClick={selectAllBooks}
+            icon={
+              <CheckCircle color={selectAll === 1 ? 'primary' : 'inherit'} />
+            }
+            label={
+              <FormattedMessage id="Select all" defaultMessage="Select all" />
+            }
+          />
+
+          <IconButton
+            css={iconButtonStyle}
+            aria-label="delete"
+            onClick={dialog}
+            icon={
+              <Delete
+                color={selectedBooks.length === 0 ? 'disabled' : 'inherit'}
+              />
+            }
+            label={<FormattedMessage id="Delete" defaultMessage="Delete" />}
+          />
+        </Right>
+      </EditBooksBar>
+    </Hidden>
+
+    <Container css={{ marginTop: '95px', marginBottom: spacing.large }}>
+      <EditBookGrid
+        books={books}
+        selectedBooks={selectedBooks}
+        active={selectAll}
+        changeActive={changeActive.bind(this)}
+        css={{ marginTop: '68px' }}
+      />
+      <Center>
+        <Button
+          onClick={onClick}
+          css={{ marginTop: spacing.large }}
+          variant="outlined"
+          size="small"
+        >
+          <FormattedMessage
+            id="Back to"
+            defaultMessage={'Back to {place}'}
+            values={{ place: favorites ? 'favorites' : 'offline library' }}
+          />
+        </Button>
+      </Center>
+      <Dialog
+        open={open}
+        onClose={dialog}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <div
+          className="content"
+          css={{
+            padding: '20px 30px'
+          }}
+        >
+          <p>
+            Are you sure you want to delete {selectedBooks.length}{' '}
```

Comment:
For å kunne ståtte flere språk så bruker vi `react-intl`, hvor man først definerer key-value-pair som man senere kan slå opp på en id for å finne tekststrengen man vi ha i appen. Det er ønskelig å gjøre det alle steder hvor vi har tekst som kan være på forskjellige språk i appen. Se `NoResults.js` for å få en ide hvordan det gjøres :) (Bare spør hvis du sitter fast eller lurer)